### DISCUSSION TITLE
New enemy creature attack

### DISCUSSION THREAD

**Discussion by IgRAzm** [FakeFishGames/Barotrauma#8418] [Ideas and suggestions]
Okay, so I have this understanding that enemies like hammerheads, molochs and threshers mostly slow down the submarine than anything; not gonna say it's inherently bad, after all they cause some direct and indirect damage to the players still, and the breach itself can be problematic if there's no rock under you or if the enemy is accompanied by other types (sadly both these scenarios are very rare in my experience).

Considering that, I think that an enemy that sucks in water and things in it en masse would be a great enemy design - this kind of attack makes breaches a significant problem, as the players near it during the attack may straight up get pulled into the hole instead of getting knocked back. This could be either a new enemy type, one that spawns together with other types more often than how it normally happens (packs of the same species), or the attack could be added to one of existing ones, like the adult molochs (for them, having their tentacles get into the hole and pull people would be neat).

**Comment by cshingle** [comment]
I think the idea of adding something else that does more than just damage the hull would be great. The Black Moloch is the most dangerous monster in the game currently in my opinion due to the fact that it can cripple a ships ability to defend it self and force players to leave the sub to kill it once they are dead in the water. I think the addition of the Giant Spineling was step in the right direction since it targets the players rather than just bang on the sub.

We all know and love the watcher. I actually feel it could use a buff based on difficulty to keep it a threat latter in the game.

Here are a few thoughts that I have:
* Something smallish that sneaks up and attaches it self to the hull to siphons power from nearby equipment. It would be a smallish threat by itself but with when combined with other monsters would be a much larger threat.
* Something smallish that has a guardian like spear that can shoot into the hull and pull people against the wall and drain their blood like a mosquito. A medic could give them blood bags to keep them from dying, but the monster would need to be killed from the outside to free the player. If the location was not in sight of a turret a diver would need to go out and remove the threat.

I really like the idea of things that the player can't just hide in the sub on the bottom and shoot at.
